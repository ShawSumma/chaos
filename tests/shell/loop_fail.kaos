3 times do
    print b
end


bool n = true

foreach n as x
end

foreach n as key : val
end
