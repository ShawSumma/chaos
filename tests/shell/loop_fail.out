Chaos Error:
File: "~/chaos/__interactive__.kaos", line 2, in <module>
print b
Undefined variable: b
Absorbed by Interactive Shell
Chaos Error:
File: "~/chaos/__interactive__.kaos", line 8, in <module>
foreach n as x
'n' is not a list!
Absorbed by Interactive Shell
Chaos Error:
File: "~/chaos/__interactive__.kaos", line 11, in <module>
foreach n as key : val
'n' is not a dictionary!
Absorbed by Interactive Shell
